### ISSUE TITLE
>500 Delta sequences with 2020 dates, mostly England/MILK-[1FF99A3..MILK-203F521] with April 2020 dates

### ISSUE THREAD

**Issue by AngieHinrichs** [COG-UK/dipi-group#146]
cog_metadata.tsv (at least, the Oct. 17th version as noted in #144 so maybe this is already fixed on your end?) has over 500 sequences assigned to Delta, but with 2020 dates.  Most are in a run of England/MILK accessions with high accession numbers but April 2020 dates:
```
grep Delta $cogUkDir/cog_metadata.csv | grep /2020 | wc -l
515

grep Delta $cogUkDir/cog_metadata.csv | grep /2020 | grep England/MILK | wc -l
511

# Lowest England/MILK- accession numbers:
grep Delta $cogUkDir/cog_metadata.csv | cut -d, -f 1,5,7 | grep /2020 | grep England/MILK | sort | head
England/MILK-1FF99A3/2020,2020-04-16,AY.4
England/MILK-1FF9A19/2020,2020-04-16,AY.4
England/MILK-1FF9A37/2020,2020-04-16,AY.4
...

# Highest England/MILK- accession numbers:
grep Delta $cogUkDir/cog_metadata.csv | cut -d, -f 1,5,7 | grep /2020 | grep England/MILK | sort | tail
...
England/MILK-203F0E4/2020,2020-04-21,AY.4
England/MILK-203F503/2020,2020-04-21,AY.4
England/MILK-203F521/2020,2020-04-21,AY.4

# Non-England/MILK- sequences:
grep Delta $cogUkDir/cog_metadata.csv | cut -d, -f 1,5,7 | g /2020 | grep -v England/MILK
England/PHEP-YYBYUTJ/2020,2020-08-03,AY.4
England/NORT-YYNW7P/2020,2020-10-27,AY.4
England/ALDP-1D32AE9/2020,2020-12-31,AY.4
England/NORW-30146BA/2020,2020-11-15,AY.5
```
April 2021 seems a lot more likely than April 2020 - typo?  

I stumbled across this when I noticed some sequences with 2020 dates on the Delta branch of the UCSC/UShER tree (>500 England/MILK's, 3 of the 4 non-MILKs, and "Scotland/CVR8617/2020": GISAID still has that name with 2020-12-27 as the sample collection date for EPI_ISL_3077206, although cog_metadata.csv now has Scotland/CVR8617/2021 with 2021-07-08).

Of the sequences also in INSDC/ENA/GenBank, it looks like GenBank has updated collection dates for most of them, but not quite all (maybe things are still propagating from ENA to GenBank?):

* NORW-30146BA (OU593375.1) still has /collection_date="2020-11-15": https://www.ncbi.nlm.nih.gov/nuccore/OU593375.1
* ALDP-1D32AE9 (OU693168.1) still has /collection_date="2020-12-31": https://www.ncbi.nlm.nih.gov/nuccore/OU693168.1
* MILK-1FF9A37: https://www.ncbi.nlm.nih.gov/nuccore/OU831021.1
* MILK-1FF9A19: OU831052.1
* MILK-1FF9B43: OU831086.1
* MILK-1FF99A3: OU831130.1
* MILK-1FFA438: OU831514.1
* MILK-1FFA131: OU831557.1
* MILK-1FFA1B9: OU831558.1
* MILK-1FFA0BC: OU832212.1
* MILK-2003E49: OU834061.1
* MILK-1FFD72D: OU834187.1
* MILK-1FFD848: OU834800.1
* MILK-1FFCA25: OU835411.1
* MILK-1FFCDC2: OU835797.1

Some Delta sequences with 2021 dates in cog_metadata.csv have 2020 dates in GenBank:
* LCST-24DE3CC has a date of 2021-05-13 in cog_metadata.csv but 2020-07-07 in GenBank: https://www.ncbi.nlm.nih.gov/nuccore/OU348289.1
* MILK-18DD450 has 2021-07-10 in cog_metadata.csv but 2020-11-04 in GenBank: https://www.ncbi.nlm.nih.gov/nuccore/OU442150.1

**Comment by rambaut** [comment]
Hi Angie, this is a known issue resulting with some data linkage issues in systems prior to CLIMB. I believe these and some others with less obvious errors are in the process of being removed from all databases. All the affected genomes were sequenced in a narrow time window in October and were likely sampled shortly prior to this so are not from April 2021. I don’t have any other information about the nature of the issue.

There are a few other Delta genomes from 2020 but these are likely sporadic reporting errors independent of the issue.

**Comment by SamStudio8** [comment]
Closing #146 as suppliers of these sequences are aware and are hopefully correcting the entries in public databases

**Comment by AngieHinrichs** [comment]
Great, thanks @rambaut and @SamStudio8.

**Comment by theosanderson** [comment]
Just to let you know @AngieHinrichs, these finally seem to have gone from GenBank so with luck they won't recur if you delete them now

**Comment by AngieHinrichs** [comment]
Thanks @theosanderson!  I get GenBank sequences and metadata from NCBI Datasets, and it turned out that they do not automatically remove records that have been removed from ENA.  But they did a one-off re-dump on request, so at last I have removed them from our tree (in local files, not yet on the main site).  I am updating the 2021-12-01 public tree -- that should hit hgwdev in a couple hours and the official download site tomorrow.
